docs/essentials/actions.md

```diff
@@ -24,6 +24,7 @@ You can use [argTypes](../api/argtypes.md) to tell Storybook that an arg to your
 <CodeSnippets
   paths={[
     'common/button-story-onclick-action-argtype.js.mdx',
+    'common/button-story-onclick-action-argtype.mdx.mdx',
```

Comment:
.mdx.mdx ?

Reply:
just seems odd :)

Reply:
@ndelangen seems a bit meta i know :P ! But i like the pattern :D